internal/security/secretstore/init.go

```diff
@@ -340,35 +340,40 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 		os.Exit(1)
 	}
 
-	cert := NewCerts(req, configuration.SecretService.CertPath, rootToken, configuration.SecretService.GetSecretSvcBaseURL(), lc)
-	existing, err := cert.AlreadyinStore()
-	if err != nil {
-		lc.Error(err.Error())
-		os.Exit(1)
-	}
+	// Uploading Kong's TLS certificate is optional if cert's path is not present
```

Comment:
See previous comment about Kong cert

Reply:
We are making this optional and deprecated eventually since there is `secrets-config proxy tls` to accomplish this.  @bnevis-i , please chime-in for more details if I am incorrect.

Reply:
There are two imminent PR's regarding removal of TLS that will follow this one.  The first is (1) don't generate TLS certificates at all (there won't be a cert to upload), (2) use "secrets-config proxy tls" to upload your BYO cert (just need to port one function call out of proxy-setup to add the missing functionality), (3) delete this code for uploading TLS certs into Vault entirely (after previous two done).

Reply:
So we're completely removing the auto-generated EdgeXCA signed TLS certificate for Kong too?

If so, IMHOP it would've been better to separate out those changes into a different PR.

Reply:
I think we are saying, at this stage, the `secrets-setup` still runs and thus generated certs for both but we are turning some default behavior to non-TLS as Vault has changed to non-TLS by default (but it does not mean that you cannot overwrite it in the docker-compose file with using environment variable `LOCAL_VAULT_CONFIG` to turn TLS back on before compose up)  so that is why this turns that into optional for now.

Reply:
@jim-wang-intel 

If we backed out this change to set these keys to the empty string
```
CertPath = "v1/secret/edgex/edgex-security-proxy-setup/kong-tls"
CaFilePath = "/tmp/edgex/secrets/ca/ca.pem"
CertFilePath = "/tmp/edgex/secrets/edgex-kong/server.crt"
KeyFilePath = "/tmp/edgex/secrets/edgex-kong/server.key"
```

would that re-enable the Kong TLS from the internal CA?

We could then put this change into the PR that deletes security-secrets-setup.

Reply:
That PR deleting security-secrets-setup is getting messier, I wouldn't want to add too much change in one PR.

Reply:
Maybe delete TLS configuration in one PR, and delete code in another?  I am trying to offer a solution that will meet both your and Tony's needs.

Reply:
@bnevis-i  Done.